### ISSUE TITLE
Endless boot/crash loop when in Repeater, Router, or Router+Client Mode

### ISSUE THREAD

**Issue by Felix-Quehl** [Xinyuan-LilyGO/LilyGo-LoRa-Series#161]
**TL;DR: `T-Beam` devices get bricked when set to `Router`, `Repeater`, or `Router+Client` mode, can only be recovered via CLI factory reset or re-flashing of the firmware.**

# Details

I have a `LilyGo T-Beam` and `T-Beam Supreme` and the device itself works fine in `client` mode. But as soon as I set it into `Router`, `Repeater`, or `Router+Client` mode it immediately gets stuck in a boot-crash-loop and becomes "bricked".

The exact root cause for this defective behavior is currently unknown, but I confidently say its probably "just" a meshtastic firmware issue - to which @attermann created a issue in the mestastic firmware repository already. As mentioned in my comment on the firmware issue, I tried all firmware version from `2.3.3` to `2.3.9` inclusively, and the defect is present in all version.

* Meshtastic Firmware Issue: https://github.com/meshtastic/firmware/issues/3066

# Unbrick Device 

To recover the device (which reverts back into client mode), I mostly had success by doing a factory reset via the meshtastic CLI:

```bash
meshtastic --factory-reset
```

But one time it was stuck so bad that i had to wipe the device and freshly flash the firmware via the web-based flashing tool:

https://meshtastic.org/docs/getting-started/flashing-firmware/esp32/web-flasher/
